src/Disks/IO/getIOUringReader.cpp

```diff
@@ -0,0 +1,41 @@
+#include <Disks/IO/getIOUringReader.h>
+
+#if USE_LIBURING
+
+#include <Interpreters/Context.h>
+#include <Common/ErrorCodes.h>
+
+namespace DB
+{
+
+namespace ErrorCodes
+{
+    extern const int LOGICAL_ERROR;
+    extern const int UNSUPPORTED_METHOD;
+}
+
+std::unique_ptr<IOUringReader> createIOUringReader()
+{
+    return std::make_unique<IOUringReader>(512);
```

Comment:
Better turn this into a constant to avoid the magic number.

Alterantively, `IOUringReader` could have a default ctor that initializes the entry count with 512.